### DISCUSSION TITLE
Adding user’s own custom model to iOS iDetection app

### DISCUSSION THREAD

**Discussion by Aoobi** [ultralytics/yolov5#4487] [Ideas]
Hi, I was wondering how hard would it be for you to update the app with a feature that would let users select a model file from their iPhone and use that model for predictions in the app?

Do you think it would take a lot of time? 

I am asking because I personally would love such feature to showcase the model, but unfortunately I don't have access to Mac to write my own app :(

**Comment by glenn-jocher** [comment]
@Aoobi yes we have some exciting improvements in our product pipelines! You will be able to train and deploy YOLOv5 models using our new HUB (https://ultralytics.com/hub), and then use our brand new app to export and preview any model you've trained to Android and iOS. We will be releasing these in the new few months, they are undergoing development and testing today.

<img width="1135" alt="Screen Shot 2021-08-23 at 3 35 28 PM" src="https://user-images.githubusercontent.com/26833433/130456716-4856b9d6-c82c-4796-94ba-ad79c683e7e6.png">
